<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::MessageHandler</class>
 <widget class="QWidget" name="GammaRay::MessageHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="KFilterProxySearchLine" name="messageSearchLine" native="true"/>
       </item>
       <item>
        <widget class="QTreeView" name="messageView"/>
       </item>
      </layout>
     </widget>
     <widget class="QTreeView" name="backtraceView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kde/kfilterproxysearchline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
